config/defaults.conf

```diff
@@ -398,6 +398,8 @@ qqp-alt_pair_attn = 1
 qqp-alt_val_interval = 1000
 qqp-alt_lr = 0.0003
 
+embeddings_train  = 0
```

Comment:
Add a comment explaining this, and move it up next to the other options that involve embeddings.

Reply:
Done!